src/components/oracle-script/OracleScriptExecuteResponse.re

```diff
@@ -52,34 +51,34 @@ let make = (~txResponse: TxCreator2.tx_response_t, ~schema: string) =>
         {switch (requestOpt) {
          | Some({result: Some(result), id}) =>
            let outputKVsOpt = Obi.decode(schema, "output", result);
-           <>
-             <div className={Css.merge([CssHelper.flexBox(), Styles.resultBox])}>
-               <div className=Styles.labelWrapper>
-                 <Text
-                   value="Output"
-                   color={theme.textSecondary}
-                   weight=Text.Regular
-                   height={Text.Px(20)}
-                 />
-               </div>
-               <div className=Styles.resultWrapper>
-                 {switch (outputKVsOpt) {
-                  | Some(outputKVs) =>
-                    <KVTable
-                      rows={
-                        outputKVs
-                        ->Belt_Array.map(({fieldName, fieldValue}) =>
-                            [KVTable.Value(fieldName), KVTable.Value(fieldValue)]
-                          )
-                        ->Belt_List.fromArray
-                      }
-                    />
-                  | None => React.null
-                  }}
+           switch (outputKVsOpt) {
+           | Some(outputKVs) =>
+             <>
+               <div className={Css.merge([CssHelper.flexBox(), Styles.resultBox])}>
+                 <div className=Styles.labelWrapper>
+                   <Text
+                     value="Output"
+                     color={theme.textSecondary}
+                     weight=Text.Regular
+                     height={Text.Px(20)}
+                   />
+                 </div>
+                 <div className=Styles.resultWrapper>
+                   <KVTable
+                     rows={
+                       outputKVs
+                       ->Belt_Array.map(({fieldName, fieldValue}) =>
+                           [KVTable.Value(fieldName), KVTable.Value(fieldValue)]
+                         )
+                       ->Belt_List.fromArray
+                     }
+                   />
+                 </div>
                </div>
-             </div>
-             <OracleScriptExecuteProof id />
-           </>;
+               <OracleScriptExecuteProof id />
+             </>
+           | None => <> <RequestFailedResult id /> </>
```

Comment:
There is a case that ResolveStatus == Success and the result can't be decoded by OBI
https://github.com/bandprotocol/cosmoscan/blob/main/src/pages/RequestIndexPage.re#L570